frontend/src/pages/MessagingQueues/MQDetails/MQDetails.style.scss

```diff
@@ -4,3 +4,42 @@
 	flex-direction: column;
 	gap: 24px;
 }
+
+.mq-overview-container {
+	display: flex;
+	padding: 24px;
```

Comment:
no light mode styles needed for this ?

Reply:
yes I need to work on lightmode, but i will be adding this in the followup to avoid conflicts